src/Form/Form.js

```diff
@@ -65,67 +66,91 @@ const Form = props => {
-        name="token"
-        placeholder="List Token"
-        value={token}
-        onChange={e => {
-          onEnterToken(e);
-        }}
-        required
-      />
-      <input
-        className={classes.inputName}
-        type="text"
-        name="name"
-        placeholder="Item Name"
-        value={name}
-        onChange={e => setName(e.target.value)}
-        required
-      />
       <div>
-        <label className={classes.inputLabel}>
+        <img
+          className={classes.groceryImg}
+          src={grocery}
+          alt="A green grocery bag full of groceries with a recycling symbol on the front. "
+        />
+      </div>
+
+      {/* <div className="input-field">
+        <input
+          type="text"
+          name="token"
+          value={token}
+          onChange={e => {
+            onEnterToken(e);
+          }}
+          required
+        />
+        <label for="token">List Token</label>
+      </div> */}
+
+      <div className="input-field">
+        <input
+          type="text"
+          name="name"
+          value={name}
+          onChange={e => setName(e.target.value)}
+          required
+        />
+        <label for="name">Item Name</label>
+      </div>
+      <div>
+        <h5>How soon do you want to buy this again?</h5>
+      </div>
+      <div className={classes.radioContainer}>
+        <label>
           <input
-            className={classes.inputFrequency}
+            required
+            className="with-gap"
             type="radio"
             name="frequency"
             value="7"
             onChange={e => setFrequency(e.target.value)}
-          />{" "}
-          Soon
+          />
+          <span className={classes.soonColor}>Soon</span>
         </label>
-        <label className={classes.inputLabel}>
+        <label>
           <input
-            className={classes.inputFrequency}
+            required
+            className="with-gap"
             type="radio"
             name="frequency"
             value="14"
             onChange={e => setFrequency(e.target.value)}
-          />{" "}
-          Kind of Soon
+          />
+          <span className={classes.kindOfSoonColor}>Kind of Soon</span>
         </label>
-        <label className={classes.inputLabel}>
+        <label>
           <input
-            className={classes.inputFrequency}
+            required
+            className="with-gap"
             type="radio"
             name="frequency"
             value="30"
             onChange={e => setFrequency(e.target.value)}
-          />{" "}
-          Not Soon
+          />
+          <span className={classes.notSoonColor}>Not Soon</span>
         </label>
       </div>
-      <input
-        className={classes.inputDate}
+
+      {/* <input
+        className="datepicker"
         type="date"
         name="date"
         value={date}
         onChange={e => setDate(e.target.value)}
-      />
-      <button>ADD</button>
+      /> */}
```

Comment:
Same here, why is this commented out? If something is commented out and is supposed to be left in, there should be a comment explaining why in the code.